src/Collectors/Callable_/AttributeCallableCollector.php

```diff
@@ -53,8 +51,9 @@ public function processNode(Node $node, Scope $scope): ?array
             }
 
             $classType = $scope->getType($classAndMethodArrayExprs->getClassExpr());
-            if ($classType instanceof ConstantStringType) {
-                $className = $classType->getValue();
+            if (count($classType->getConstantStrings()) === 1) {
```

Comment:
PHPStan 2.0 throws an error if you use this `instanceof ConstantStringType` construction and they recommend that you use `->getConstantStrings()` instead, see https://phpstan.org/blog/why-is-instanceof-type-wrong-and-getting-deprecated

Reply:
can/could should we return multipe results when `count($classType->getConstantStrings()) > 1` instead of discarding it?

Reply:
This is the class passed to a `Symfony\Component\Validator\Constraints\Callback` attribute instance. It cannot be multiple classes

Reply:
I see :+1: - for attributes it makes sense.